infrastructure/cli/integration/golden/cli.yaml

```diff
@@ -1,2 +1,2 @@
-apihost: http://localhost:3001
+apihost: http://localhost:64789
```

Comment:
not really a relevant change... the way it works now this is bound to happen. I think the best shot for us here is to use the init command and generate a file for the current test (but we need to improve init so that it can take a dir/file argument)